tests/Wanchain/TWCoinTypeTests.cpp

```diff
@@ -15,7 +15,7 @@
 
 TEST(TWWanchainCoinType, TWCoinType) {
     auto symbol = WRAPS(TWCoinTypeConfigurationGetSymbol(TWCoinTypeWanchain));
-    auto txId = TWStringCreateWithUTF8Bytes("123");
+    auto txId = TWStringCreateWithUTF8Bytes("0x180ea96a3218b82b9b35d796823266d8a425c182507adfe5eeffc96e6a14d856");
```

Comment:
It's OK, just a comment:  for the sake of the test any txId would do, using a real one is the best, but 123 was also fine.